Sentinel/Classes/Core/PreferencesInfo/Items/PreferenceToggleItem.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  PreferenceToggleItem.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 17.03.2025..
+//
+
+import Foundation
+
+@available(*, deprecated, renamed: "PreferenceToggleItem", message: "ToggleToolItem has been renamed to PreferenceToggleItem, and please use the new name, as it is more consistent with other preference items")
```

Comment:
This part can be removed.
> and please use the new name, as it is more consistent with other preference items

I would rather add something like; Usage of the `ToggleToolItem` is not recommended due to the plan for deprecation.